xml/System.IO.Enumeration/FileSystemEnumerator`1.xml

```diff
@@ -312,9 +312,9 @@
         <Parameter Name="entry" Type="System.IO.Enumeration.FileSystemEntry" RefType="ref" />
       </Parameters>
       <Docs>
-        <param name="entry">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="entry">A file system entry reference.</param>
+        <summary>Allows indicating if a file system entry should be recursed or not.</summary>
+        <returns><see langword="true" /> if the directory entry given should be recursed into, <see langword="false" /> otherwise.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the specified directory entry should be recursed into; otherwise, <see langword="false" />.</returns>
```